src/clients/payment/create/index.ts

```diff
@@ -1,5 +1,6 @@
 import { RestClientApi } from "@utils/types";
+import { CreateResponse, CreatePayload } from "./types";
 
 export default (restClient: RestClientApi) => {
-    return () => {}
+    return async (data: CreatePayload): Promise<CreateResponse> => await restClient<CreateResponse>(`/api/v1/payment`, { method: "POST", body: JSON.stringify(data) });
```

Comment:
an idea is to move JSON.stringify to rest client
and you can short this?
(restClient) => (data) => restClient<CreateResponse>
use typescript inference too